README.md

```diff
@@ -18,7 +18,7 @@ You can run the game with:
 ```commandline
 PORT=<PORT> python app.py
 ```
-Configurations can be set in `config.py` (not part of the repo as it contains secrets).
+Configurations should be set in a `semantle.cfg` file, following the example `semantle.cfg.format`.
```

Comment:
it's actually _can_ and not _should_, as you can pass the entire configuration through env var `HS_CONFIG`